client/java/generator/src/main/java/io/openlineage/client/Generator.java

```diff
@@ -55,10 +55,30 @@ public static void generate(String baseURL, InputStream input, File output) {
     try {
       ObjectMapper mapper = new ObjectMapper();
       JsonNode schema = mapper.readValue(input, JsonNode.class);
+      if (schema.has("$id") && schema.get("$id").isTextual()) {
```

Comment:
_Minor_: Do we need to throw an exception if the node is not textual, or is it ok to just skip / continue?